index.html

```diff
@@ -4,8 +4,82 @@
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>project 3</title>
+     <link rel="stylesheet" href="style.css">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Kanit:wght@300&display=swap" rel="stylesheet">
  </head>
  <body>
-     
+     <header>
+       <h1>
+        Design conference<br>
+         ——Typography In Web Design
+       </h1>
+      </header>
+       <nav>
+      <ul class="navigation" >  
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Speakers</a></li>
+      <li><a href="#">Register</a></li>
+      <li><a href="#">Contact</a></li>
+      </ul> 
+      </nav>
+   <main>
+     <ul class="u-list-reset ">
+    <li class="talk1">
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2F1.png?v=1607879901347"
+        alt="Photo of Jordan Devos" />
```

Comment:
Screen readers already tell folks that this is an image, so your `alt` values here can be more descriptive _and_ eliminate the "photo of...".

Henny's project has some great `alt` text if you want some inspiration: https://github.com/Henny-2020/Project3/blob/78936b2c4dbe9ae0fd434c6c09078f253c425ffd/index.html

Or you can review my notes on [writing good `alt` text](https://mica-web.github.io/learn/html/images#writing-good-alt-text).